ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule.hs

```diff
@@ -220,35 +210,6 @@ instance Condense Tick where
     where
       showDT t = printf "%.6f" (realToFrac t :: Double)
 
-tickDefault :: PointScheduleConfig -> Word -> Peer NodeState -> Tick
```

Comment:
Is the datatype `Tick` unused now?

Is the datatype `PointScheduleConfig` unused now?

Reply:
You might very well be right. Let me see if I can get rid of them.

Reply:
You were absolutely right. Done in 6b5de2661ba6d2a9774ee575f4f85d0d7385e90f.